client/server/middleware/unsupported-browser.js

```diff
@@ -55,7 +55,7 @@ function allowPath( path ) {
 		? path.replace( new RegExp( `^/${ prefixedLocale }` ), '' )
 		: path;
 
-	const allowedPaths = [ '/browsehappy', '/themes', '/theme' ];
+	const allowedPaths = [ '/browsehappy', '/themes', '/theme', '/calypso/evergreen' ];
```

Comment:
The entire `/calypso` directory should be excluded because there are also paths like `/calypso/images/gridicons.svg` or `/calypso/reader-mobile.css`. See your local `public/` directory after a production build. All its contents are accesible as `/calypso/*`.

Reply:
Nevermind, I just discovered #59792 🙂